Formula/osh.rb

```diff
@@ -1,8 +1,8 @@
 class Osh < Formula
   desc "Two ports of /bin/sh from V6 UNIX (circa 1975)"
   homepage "http://v6shell.org"
-  url "http://v6shell.org/src/osh-20160108.tar.gz"
-  sha256 "51980473429cd1f8dad255c30749b8588278caf42b28185e22b9455e8f381bf7"
+  url "http://v6shell.org/src/osh-4.2.1.tar.gz"
```

Comment:
Actually I was about to ask about this. I was not able to upgrade locally, I had to remove and install new version. Can we somehow handle a case like this?